### PR TITLE
Upgraded to Spring Boot 2.2

### PR DESCRIPTION
**Pull request by Buzzardo**
I upgraded to Spring Boot 2.2, which necessitated upgrading to JUnit 5 and replacing the Gradle wrappers.

### CODE REVIEW

File: complete/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,5 @@
-#Thu Mar 01 09:04:46 CST 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.2.1-bin.zip
```
Comment (sbrannen) [inline complete/gradle/wrapper/gradle-wrapper.properties]: Why not upgrade to Gradle 5.6.3?

File: complete/src/test/java/com/example/accessingdatarest/AccessingDataRestApplicationTests.java [hunk 2]
```diff
@@ -16,9 +16,9 @@
 
 package com.example.accessingdatarest;
 
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.runner.RunWith;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
```
Comment (sbrannen) [inline complete/src/test/java/com/example/accessingdatarest/AccessingDataRestApplicationTests.java]: This import is now unused -- right?

File: complete/gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (Buzzardo) [reply complete/gradle/wrapper/gradle-wrapper.properties]: 5.2.1 is what I had installed. Spring Boot currently requires 4.10.

Comment (sbrannen) [reply complete/gradle/wrapper/gradle-wrapper.properties]: Ahhh.... Well, the goal of the [Gradle Wrapper](https://docs.gradle.org/current/userguide/gradle_wrapper.html) is that you specify a version of Gradle used to build the project, regardless of what version of Gradle is installed on the user's machine. Then, when the user executes `./gradlew ...`, the wrapper will download the specified version of Gradle.

So, for a demo project, I pretty much always go with the latest and greatest version of Gradle, but in the end it's up to you.

**Pull request merged**